src/components/LoadingBar/LoadingBar.tsx

```diff
@@ -0,0 +1,50 @@
+import { useStyles2 } from '@grafana/ui';
+import React, { CSSProperties } from 'react';
+
+import { getStyles } from './LoadingBar.styles';
+
+/**
+ * Properties
+ */
+export interface LoadingBarProps {
+  /**
+   * Width
+   */
+  width: number;
+
+  /**
+   * Label
+   */
+  ariaLabel?: string;
+}
+
+/**
+ * Constants
+ */
+const msPerPixel = 2.4;
+const minDurationMs = 500;
+const maxDurationMs = 4000;
+
+/**
+ * Loading Bar
+ * Not available in Grafana 9
```

Comment:
@vitPinchuk We don't support Grafana 9 and we can use LoadingBar from Grafana UI. Needs to remove from Business Forms as well.